src/common/app_metrics.cpp

```diff
@@ -133,6 +133,8 @@ void buddy::metrics::RecordRuntimeStats() {
 
     METRIC_DEF(heap, "heap", METRIC_VALUE_CUSTOM, 503, METRIC_HANDLER_ENABLE_ALL);
     metric_record_custom(&heap, " free=%zui,total=%zui", xPortGetFreeHeapSize(), static_cast<size_t>(heap_total_size));
+
+    RecordUptime();
```

Comment:
1. Adding a metric is a separate thing from adding a statistics item, it should be in a separate commit
2. We use `snake_case` for functions
3. No other metric in the RecordRuntimeStats has a dedicated record function